src/test/java/com/rocketden/main/UserTests.java

```diff
@@ -126,4 +130,33 @@ public void createNewUserNicknameContainsSpaces() throws Exception {
 
         assertEquals(ERROR.getResponse(), actual);
     }
+
+    @Test
+    public void deleteExistingUser() throws Exception {
+        CreateUserRequest createRequest = new CreateUserRequest();
+        createRequest.setNickname("rocket");
+        
+        DeleteUserRequest request = new DeleteUserRequest();
+        request.setNickname("rocket");
+
+        this.mockMvc.perform(post(POST_USER)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .content(Utility.convertObjectToJsonString(request)));
+
+        this.mockMvc.perform(delete(POST_USER)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .content(Utility.convertObjectToJsonString(request)))
+                .andExpect(status().isAccepted());
```

Comment:
nit: I would suggest doing something similar to the above tests, where it gets the returned objects and verifies that they match up correctly.